kivy/utils.py

```diff
@@ -113,14 +113,36 @@ def rgba(s, *args):
 def get_color_from_hex(s):
     '''Transform a hex string color to a kivy
     :class:`~kivy.graphics.Color`.
+
+    .. versionadded:: 2.1.0
```

Comment:
The function `get_color_from_hex` has been added a quite long time ago (Kivy doesn't even had a version at that time).

Instead, is fine to use a `.. versionchanged:: 2.2.0`, as you added support for something new.

Reply:
I do not know how to use documentation for kivy properly. Thanks for the helpfully advice.